addons/account/tests/test_fiscal_position.py

```diff
@@ -154,3 +154,14 @@ def test_20_fp_one_tax_2m(self):
         mapped_taxes = self.fp2m.map_tax(self.src_tax)
 
         self.assertEqual(mapped_taxes, self.dst1_tax | self.dst2_tax)
+
+    def test_30_fp_country_delivery(self):
+        def assert_fp(partner, delivery, expected_pos, message):
+            self.assertEqual(
+                self.fp.get_fiscal_position(partner.id, delivery.id).id,
+                expected_pos.id,
+                message)
```

Comment:
Do not copy paste

Reply:
Did it fast. Inspired by your own code :-)